frameworks/Java/undertow/undertow-base.dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM java:latest
```

Comment:
One thing I'd comment on about the implementation of docker in FWBM so far is that I'd much prefer that the images were named `techempower/java` instead of just `java`. There already exists a `java:latest` image in the docker hub, which I can imagine is confusing to implementers. You also might accidentally end up pulling the docker hub java image and changing the build environment.

Reply:
It is on the [roadmap](https://github.com/TechEmpower/FrameworkBenchmarks/issues/3296)

Reply:
Where?